requirements.txt

```diff
@@ -43,11 +49,14 @@ pyusb==1.0.2
 qrcode==6.1
 reportlab==3.5.13; python_version < '3.8'
 reportlab==3.5.55; python_version >= '3.8'
-requests==2.21.0
+requests==2.21.0; python_version <= '3.9'
+requests==2.25.1; python_version > '3.9'  # (Jammy) versions < 2.25 aren't compatible w/ urllib3 1.26. Bullseye = 2.25.1. min version = 2.22.0 (Focal)
+urllib3==1.26.5; python_version > '3.9'  # (Jammy) indirect / min version = 1.25.8 (Focal with security backports)
 zeep==3.2.0
 python-stdnum==1.8
 vobject==0.9.6.1
-Werkzeug==0.16.1
+Werkzeug==0.16.1 ; python_version <= '3.9'
+Werkzeug==2.0.2 ; python_version > '3.9'  # (Jammy)
```

Comment:
is this a requirement? aren't older versions of werkzeug still compatible with 3.10?

Reply:
I think it's better to match the distro version.
e.g.: if someone (or the runbot ;-) ) tries to pip install the `requirements.txt` on a Jammy distribution, without this line, the werkzeug package will be downgraded. Probably not what that user expects.

And that will not interfere on older distros.